api-common-java/.github/release-please.yml

```diff
@@ -1,24 +0,0 @@
-releaseType: java-yoshi
-bumpMinorPreMajor: true
-handleGHRelease: true
-branches:
-  - releaseType: java-lts
-    bumpMinorPreMajor: true
-    branch: 1.10.1-sp
-    handleGHRelease: true
-  - bumpMinorPreMajor: true
-    handleGHRelease: true
-    releaseType: java-yoshi
-    branch: java7
-  - releaseType: java-backport
-    bumpMinorPreMajor: true
-    handleGHRelease: true
-    branch: 2.0.x
-  - releaseType: java-backport
-    bumpMinorPreMajor: true
-    handleGHRelease: true
-    branch: 2.1.x
-  - releaseType: java-backport
-    bumpMinorPreMajor: true
-    handleGHRelease: true
-    branch: 2.2.x
```

Comment:
How are we now handling LTS for these `api-common-java` versions?

Reply:
We will keep using the LTS branches in the old `api-common-java` repo.